core/src/main/scala/fr/davit/akka/http/metrics/core/HttpMetricsSettings.scala

```diff
@@ -51,12 +52,18 @@ trait HttpMetricsSettings {
     */
   def includeStatusDimension: Boolean
 
+  /**
+    * Include custom labels, using the Dimension object
+    */
+  def includeCustomLabels: Seq[CustomDimension]
```

Comment:
naming is inconsistent with other settings. here we don't have the `include` boolean. I would prefer a naming like `serverDimensions`, returning `Dimension`

Reply:
Yes, thank you. The naming was... questionable ;)